static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,848 @@
+$header-height: 48px;
+$thumb-width: 90px;
+$badge-width: 80px;
+
+.hosted-page {
+    @include f-textSans;
+}
+
+.hosted__glogo {
+    display: inline-block;
+    padding: $gs-baseline/3 $gs-gutter/2;
+    background-color: $guardian-brand;
+    text-align: right;
+}
+
+.hosted__guardian-logo__svg {
+    fill: #ffffff;
+    width: 115px;
+    height: 40px;
+}
+
+.hosted__glogo .inline-logo {
+    display: inline-block;
+    margin-top: -5px;
+}
+
+.hosted__glogotext {
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: .5px;
+    line-height: 16px;
+    margin: 0;
+    text-align: left;
+    color: rgba(255, 255, 255, .5);
+}
+
+.hostedbadge {
+    position: absolute;
+}
+
+.hostedbadge__logo.hostedbadge__logo {
```

Comment:
Just checking, is this intenstional? If a element has the same classname twice, do this thing?